python/tests/unit/arcticdb/version_store/test_append.py

```diff
@@ -280,3 +282,173 @@ def test_(initial: InputFactories, append: InputFactories, match, swap, lmdb_ver
     to_append, _ = append.make(abs(next_start), 1)
     with pytest.raises(NormalizationException):
         lib.append("s", to_append, match=match)
+
+
+@pytest.mark.parametrize(
+    [
+        "segment_count",
+        "col_per_append_df",
+        "col_name_set",
+        "num_of_segments_to_compact_threshold",
+        "num_of_repeat",
+        "num_of_compact_per_cycle",
+        "lmdb_version_store_column_buckets",
+        "lmdb_version_store_tiny_segment",
+        "lmdb_version_store_tiny_segment_dynamic_string",
+        "lmdb_version_store_tiny_segment_dynamic",
+        "lmdb_version_store_tiny_segment_dynamic_dynamic_string",
+        "lmdb_version_store_column_buckets_dynamic_string",
+    ],
+    [(4, 20, 100, 1, 5, 2, True, True, True, True, True, True)],
+    indirect=[
+        "lmdb_version_store_column_buckets",
+        "lmdb_version_store_tiny_segment",
+        "lmdb_version_store_tiny_segment_dynamic_string",
+        "lmdb_version_store_tiny_segment_dynamic",
+        "lmdb_version_store_tiny_segment_dynamic_dynamic_string",
+        "lmdb_version_store_column_buckets_dynamic_string",
+    ],
+)
```

Comment:
Using test fixtures in this style is unnecessarily opaque, and borderline unreadable.

Please just inject the `version_store_factory` pass the necessary config combinations.